DotNetHowToSynonyms/DotNetHowToSynonyms/DotNetHowToSynonyms.csproj

```diff
@@ -0,0 +1,89 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{001AB02D-B165-4FB4-808C-48502C69E8BA}</ProjectGuid>
```

Comment:
Although currently this project lives in its own solution, you should change its ProjectGuid just in case we decide to combine it in a single solution with DotNetHowTo later. Each .csproj file should have a unique ProjectGuid. You'll need to change the reference in the .sln file too.